draft-ietf-quic-transport.md

```diff
@@ -1876,7 +1876,9 @@ Connection establishment implicitly provides address validation for both
 endpoints.  In particular, receipt of a packet protected with Handshake keys
 confirms that the client received the Initial packet from the server.  Once the
 server has successfully processed a Handshake packet from the client, it can
-consider the client address to have been validated.
+consider the client address to have been validated.  Servers MAY treat the
+receipt of a packet using a server generated destination connection ID with
```

Comment:
nit: s/server generated/server-generated/

Reply:
SGTM, for the future, suggestions are faster for me to commit.